views/forms/TextInputTopLabel/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { InputLabel, FormHelperText, OutlinedInput } from '@mui/material/'
+import { useController } from 'react-hook-form'
+
+import { lineHeight, fontSize, borderRadius } from '../../../constants'
+
+const TextInputTopLabel = (props) => {
+  const {
+    field,
+    formState: { errors },
+  } = useController(props)
+
+  return (
+    <>
+      <InputLabel
+        {...props?.inputLabel}
+        aria-label={props.label}
+        shrink
+        sx={{
+          color: 'text.primary',
+          fontWeight: 500,
+          lineHeight: lineHeight[21],
+          fontSize: fontSize[18],
+          ...props.inputLabel?.sx,
+        }}
+        required={props.required}
+        htmlFor={props.name}
+      >
+        {props.label}
+      </InputLabel>
+      <OutlinedInput
+        {...field}
+        aria-invalid={!!errors[props.name] ? 'true' : 'false'}
+        id={props.name}
+        fullWidth={props.fullWidth}
+        error={!!errors[props.name]}
+        inputProps={{ ...props.inputProps }}
+        margin={props.margin || 'none'}
+        required={props.required}
+        type={props.type}
+        sx={{ borderRadius: borderRadius[8], ...props.sx }}
+        size={props.size}
+        disabled={props.disabled}
+      />
+      {!!errors[props.name] && (
+        <FormHelperText
+          aria-label={`${props.name}-error`}
```

Comment:
do we need this? Screen readers can read the error message below.

Reply:
[2cb019c](https://github.com/AMP-SCZ/dpdash/pull/478/commits/2cb019cb99525d69d3475ad8700252c8c896ccb7)